docs/source/slurmguide/slurmmonitoringandmanaging.md

```diff
@@ -1,33 +1,20 @@
 (slurm-monitoring-and-managing)=
 # Monitoring and Managing Jobs
 
-The monitoring and management section aims to give you a solid understanding of how to manage and control your jobs effectively. Always ensure to monitor your jobs regularly and adjust parameters as needed to achieve the best performance.
+This section will explain the basics of checking the status of the cluster, checking for the details on the compute nodes, and checking on the running jobs you have on the cluster.
 
 (cluster-and-node-states)=
-## Cluster and Node States: `sinfo`
-Below are some more examples of using `sinfo` and `scontrol` to provide information about the state of the cluster and specific nodes.
+## Cluster and Node States
```

Comment:
These headings should match better with the description under Monitoring and Managing Jobs.

Reply:
This heading also seems unnecessary.